packages/compass-components/package.json

```diff
@@ -46,6 +46,7 @@
     "@leafygreen-ui/hooks": "^7.3.3",
     "@leafygreen-ui/icon": "^11.21.0",
     "@leafygreen-ui/icon-button": "^15.0.3",
+    "@leafygreen-ui/info-sprinkle": "^1.0.0",
```

Comment:
TIL
Probably a lot of other places in Compass we can look at updating to use this over time.